src/js/controllers/disclaimer.js

```diff
@@ -32,6 +36,8 @@ angular.module('copayApp.controllers').controller('disclaimerController',
       });
     };
 
-    create();
-
+    if (!profileService.profile)
+      create();
+    else
+      applicationService.restart();
```

Comment:
Yes. That problem not solved yet. I will make a new file that includes only the Terms of Use and not the icon and "welcome to copay" title.